cas-server-support-saml/src/main/java/org/jasig/cas/support/saml/web/view/Saml10SuccessResponseView.java

```diff
@@ -96,7 +96,6 @@ protected void prepareResponse(final Response response, final Map<String, Object
      */
     private Map<String, Object> prepareSamlAttributes(final Map<String, Object> model, final Service service) {
         final Map<String, Object> authnAttributes = new HashMap<>(getAuthenticationAttributesAsMultiValuedAttributes(model));
```

Comment:
Yes, the result of `getAuthenticationAttributesAsMultiValuedAttributes` is wrapped and there is no issue with that. But I just removed line below which seemed like a mistake to me:
`new HashMap<>(getAuthenticationAttributesAsMultiValuedAttributes(model));`

Could you please explain to me where I am wrong here?